README.md

```diff
@@ -2,9 +2,9 @@
 
 <img align="right" src="https://app.configu.com/logo192.png" height="150px" alt="configu logo">
```

Comment:
We should make this a link to the website.

```suggestion
<a href="https://configu.com" target="_blank" rel="noopener noreferrer"><img align="right" src="https://app.configu.com/logo192.png" height="150px" alt="configu logo"></a>
```

Reply:
Done without the rel